packages/@aws-cdk/aws-iam/test/role.test.ts

```diff
@@ -4,6 +4,36 @@ import { Duration, Stack, App, CfnResource, RemovalPolicy, Lazy, Stage, DefaultS
 import { Construct } from 'constructs';
 import { AnyPrincipal, ArnPrincipal, CompositePrincipal, FederatedPrincipal, ManagedPolicy, PolicyStatement, Role, ServicePrincipal, User, Policy, PolicyDocument, Effect } from '../lib';
 
+describe('isRole', () => {
```

Comment:
```suggestion
describe('isRole() returns', () => {
```

Reply:
Thank you for your nice suggestion!

I've commited this!!